core/components/atoms/heading/heading.js

```diff
@@ -2,9 +2,11 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { colors, fonts } from '@auth0/cosmos-tokens'
+import margin from '../../_helpers/margin'
 
 const BaseHeading = styled.h1`
   margin: 1em 0;
```

Comment:
Since we are offering it as an opt-in, can we remove this default margin?

Reply:
We can remove top margin as long as we give them an easy to keep their visual styles

Either with this margin API or by creating smart layout components which bake in the defaults (cc @landitus)